internals/scripts/clean.js

```diff
@@ -37,16 +37,32 @@ cp('internals/templates/homePage.js', 'app/components/HomePage/index.js');
 rm('-rf', 'app/sagas/*');
 cp('internals/templates/sagas.js', 'app/sagas/index.js');
 
-// Cleanup selectors folder
-rm('-rf', 'app/selectors/*');
+// Cleanup selectors folder, leave selectLocationSelection(.test).js
+rm('-f', [
+  'app/selectors/tests/currentUserSelector.test.js',
+  'app/selectors/tests/errorSelector.test.js',
+  'app/selectors/tests/globalSelector.test.js',
+  'app/selectors/tests/loadingSelector.test.js',
+  'app/selectors/tests/reposSelector.test.js',
+  'app/selectors/tests/routeSelector.test.js',
+  'app/selectors/tests/usernameSelector.test.js',
+  'app/selectors/currentUserSelector.js',
+  'app/selectors/errorSelector.js',
+  'app/selectors/globalSelector.js',
+  'app/selectors/loadingSelector.js',
+  'app/selectors/reposSelector.js',
+  'app/selectors/routeSelector.js',
+  'app/selectors/usernameSelector.js',
+]);
```

Comment:
This is probably a bad idea because people might play with the example and add new selectors, which won't be deleted this way. I think deleting the whole folder and having a template for the location selector (and whatever else selector you need) is a better way to go here!